app/controllers/concerns/hyku_addons/users_controller_behavior.rb

```diff
@@ -12,7 +12,15 @@ def index
 
     def show
       @user = User.find_by(email: params[:email], display_profile: true)
```

Comment:
Is `@user` used in the view? If not, it could be a normal method variabe.

Reply:
https://github.com/ubiquitypress/hyku-api/blob/main/app/views/hyku/api/v1/users/show.json.jbuilder